oasislmf/model_execution/bin.py

```diff
@@ -151,6 +155,18 @@ def prepare_run_directory(
                     os.symlink(path, os.path.join(model_data_dst_fp, fn))
             except Exception:
                 shutil.copytree(model_data_fp, os.path.join(model_data_dst_fp, fn))
+
+        if user_data_dir and os.path.exists(user_data_dir):
+            for path in glob.glob(os.path.join(user_data_dir, '*')):
+                fn = os.path.basename(path)
+                try:
+                    if os.name == 'nt':
+                        shutil.copy(path, os.path.join(oasis_dst_fp, fn))
+                    else:
+                        os.symlink(path, os.path.join(oasis_dst_fp, fn))
+                except Exception:
```

Comment:
That's quite a generic `except`. If you really don't want to be more specific consider a comment explaining why `copytree` is expected to work when `copy` and `symlink` didn't.